src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -357,19 +358,38 @@ private void installIncomingPacketReader(DatagramSocket socket)
         // enough (it'll only be a bit of the DTLS path) that running it in the
         // IO pool is fine
         TaskPools.IO_POOL.submit(() -> {
-            byte[] bbuf = ByteBufferPool.getBuffer(1500);
-            DatagramPacket p = new DatagramPacket(bbuf, 0, 1500);
+
+            // We need this buffer to be 1500 bytes because we don't know how
+            // big the received packet will be. But we don't want to allocate
+            // large buffers for all packets.
+            byte[] receiveBuf = new byte[1500];
```

Comment:
it doesn't seem to me like this precludes using the pool for this buffer.  we will just grab it once (and return it once when this entire thing exits) but we can still get a 1500 byte buffer from the pool, that way we can maintain the invariant that all packet buffers come from the pool, which would be useful.